backends/edk2-compat/edk2-svc-write.c

```diff
@@ -5,127 +5,135 @@
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
-#include <stdlib.h>// for exit
+#include <stdlib.h> // for exit
 #include <argp.h>
 #include "secvarctl.h"
-#include "backends/edk2-compat/include/edk2-svc.h"// import last!!
+#include "backends/edk2-compat/include/edk2-svc.h" // import last!!
 
-
-static int updateSecVar(const char *var, const char *authFile, const char *path, int force);
+static int updateSecVar(const char *var, const char *authFile, const char *path,
+			int force);
 
 struct Arguments {
 	int helpFlag, inpValid;
 	const char *pathToSecVars, *varName, *inFile;
-}; 
+};
 static int parse_opt(int key, char *arg, struct argp_state *state);
 
-
 /*
  *called from main()
  *handles argument parsing for write command
  *@param argc, number of argument
  *@param arv, array of params
  *@return SUCCESS or err number
 */
-int performWriteCommand(int argc, char* argv[])
+int performWriteCommand(int argc, char *argv[])
 {
 	int rc;
-	struct Arguments args = {	
-		.helpFlag = 0, .inpValid = 0, 
-		.pathToSecVars = NULL, .inFile = NULL, .varName = NULL
-	};
+	struct Arguments args = { .helpFlag = 0,
+				  .inpValid = 0,
+				  .pathToSecVars = NULL,
+				  .inFile = NULL,
+				  .varName = NULL };
```

Comment:
Hmm, not a huge fan of what the clang-format file is doing to these struct initializers, but I think I'll survive.

Reply:
it's not great hey. having said that, I don't care that much either; if there's a tool that'll do it automatically for me I'll tolerate quite a bit.

Reply:
Ya not seeing anything in the `.clang-format` file that we can use to our advantage on this. Only thing I see in the docs is `AfterStruct` which just puts the opening brace on a different line.

Reply:
Out of curiosity, does it complain if you put a newline after the opening `{`, and before the closing `}` like so:
```
	struct Arguments args = {
				  .helpFlag = 0,
				  .inpValid = 0,
				  .pathToSecVars = NULL,
				  .inFile = NULL,
				  .varName = NULL
	};
```

I swear I saw somewhere else where it didn't complain about it, maybe it was the list of structs initialization.